plugins/inventory-backend/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "target": "es5",
     "module": "commonjs",
     "esModuleInterop": true,
+    "lib": ["es2019"],
```

Comment:
this lets us use `Array.flat()`

Reply:
Fits with the compilation target I'm aiming for with https://github.com/spotify/backstage/pull/655